### DISCUSSION TITLE
Add the ability to invoke the Windows 11 Quick Actions panel when using Windows 10 Taskbar Style

### DISCUSSION THREAD

**Discussion by Technostud** [valinet/ExplorerPatcher#2662] [Ideas]
When using the Windows 10 Taskbar Style, is there a way to invoke the Windows 11 Quick Actions panel?  I understand if not, but otherwise, my suggestion is to add a way to invoke the Windows 11 Quick Actions panel/flyout when using the Windows 10 Taskbar Style.

Please see https://www.xda-developers.com/use-quick-actions-windows-11/ for pictures of the Windows 11 Quick Actions panel.

In addition to other settings, I like adjusting my laptop's screen brightness from Windows 11's Quick Access Panel.  When using Windows 10 Taskbar Style, I can't figure out how to do this.

I'm sorry if I've missed other ways to do all of this when using the Windows 10 Taskbar Style,  Please let me know.

**Comment by pyrates999** [comment]
I think quick actions is part of the windows 11 taskbar only.  Pressing windows key + a brings up the notifications menu for windows 10.

So try that keyboard shortcut to see if you see quick actions while using the windows 10 taskbar in windows 11.

Laptop screen brightness as well as volume is usually controlled with a fn key and the function keys f1-f12.  It is on my laptop at least.

**Reply by Technostud** [reply]
Thank you!  That keyboard shortcut is a good workaround:  Windows Key + A.  Thanks again!

I often remote control my laptop from another computer, so the fn key is not an option for me in that circumstance.  I can always go the Settings\System\Display and change my brightness there too, but now Windows Key + A will be faster.

Especially seeing that Quick Actions can be invoked with Windows Key + A, it seems that ExplorerPatcher could add the option to open the Windows 11 Quick Actions Panel when clicking on the speaker, battery, or network icons in the tray.  Just an idea.

**Reply by pyrates999** [reply]
lets see what the developers have to say then.

**Comment by tallman69** [comment]
I am using the Windows 10 taskbar on Windows 11, and I have a gear icon in between the speaker icon and the clock on my taskbar that opens what you are calling the Quick Action Panel.  If I hover the pointer over the gear, it is called the Control Center.  You can show or hide the Control Center icon through the ExplorerPatcher Properties app. With the Windows 10 taskbar set as the taskbar style, go to the System tray tab, there the Control Center button can be toggled on and off.
